src/features/login/actions.js

```diff
@@ -39,11 +39,30 @@ const getLoginLocationsFailed = (message) => ( {
   }
 } );
```

Comment:
I should have thoughts of this design up-front, but the logos are not part of the login functionality at all, only the header... so we should create a new directory under "features" called "header",  and these new actions, reducers and sagas should go in new actions.js, reducers.js and sagas.js in that new subdirectory.

Then the action creators should be "getHeaderLogoLinks" and "HEADER_TYPES.LOGO_LINKS.REQUESTED", etc...

The new reducer should be set up in a new section of the store called "header", and then then mapping in Header.jsx could be changed accordingly.  That is we'd add an entry **"header: headerReducers"** here:

https://github.com/openmrs/openmrs-react-components/blob/master/src/index.js#L41